site/pages/pledge/[address].tsx

```diff
@@ -10,8 +12,16 @@ export const getStaticProps: GetStaticProps = async (ctx) => {
   const translation = await loadTranslation(ctx.locale);
   const { address } = ctx.params as { address: string };
   let pledge;
+  let resolvedAddress;
+
+  const isDomainInURL = getIsDomainInURL(address);
+  const domain = isDomainInURL ? address : null;
 
-  if (!ethers.utils.isAddress(address)) {
+  try {
+    resolvedAddress = isDomainInURL
+      ? await getAddressByDomain(address)
```

Comment:
@Atmosfearful
This has got me stumped, exceptions from a non ens/kns domain in the url aren't being caught and redirected as expected. Any ideas on this one?

<img width="557" alt="image" src="https://user-images.githubusercontent.com/97446324/178515511-3a4f4a22-7184-400d-9814-3343ca87afef.png">

Reply:
Ah I think I worked it out. I just needed to set an Infura ID ☺️

Reply:
Yes exactly! 
This error should only appear when you are trying to resolve an ENS address without the INFURA ID.

But you said above that this error happened also on non ENS addresses? Or did I got that wrong?